### PR TITLE
lets see

### PR DESCRIPTION
**Pull request by Havrysh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -29,20 +30,29 @@ const alex = {
...
     <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
+      <Person
+        namE={misha.name}
+        age={misha.age}
+        partner={misha.partnerName}
+        sex={misha.sex}
+      />
```
Comment (marina-tilniak) [inline src/App.js:38]:
```suggestion
      <Person {...misha} />
```

Comment (marina-tilniak) [reply src/App.js:38]: так як ці змінні тобі тут не потрібні можна передавати об'єкт і робити деструктуризацію в компоненті Person

**Review by marina-tilniak — CHANGES_REQUESTED** [review]
додай Person.jsx

File: src/components/Person.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <p className="Person__age">{`I am ${age}`}</p>
+    ): null}
+    {sex === 'm' ? (
+      <p className="Person__partner">
+        {partnerName ? `My wife's name is ${partnerName}` : 'I am not married'}
+      </p>
+    ) : (
+      <p className="Person__partner">
+        {partnerName ? `My husbands's name is ${partnerName}` : 'I am not married'}
+      </p>
```
Comment (marina-tilniak) [inline src/components/Person.jsx:23]: same here

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: src/components/Person.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  name,
+  age,
+  partnerName,
+  sex,
+}) => (
+  <>
+    <h2 className="Person__name">{name}</h2>
+    {age ? (
+      <p className="Person__age">{`I am ${age}`}</p>
+    ): null}
```
[the remark this answers is not in the document]
Comment (Havrysh) [reply src/components/Person.jsx:15]: Got you, wrote this way bcs saw it in Mykhailo's video

[the remark this answers is not in the document]
Comment (marina-tilniak) [reply src/components/Person.jsx]: I think it's more readable option and we use it in our projects, but in your future company you could find your option and it's also ok

**Review by marina-tilniak — APPROVED** [review]

**Pull request closed without merging**